corehq/apps/userreports/tests/test_pillow.py

```diff
@@ -197,11 +197,9 @@ def test_bad_spec_error(self, _):
         }
         data_source_1.save()
         del ExpressionFactory.spec_map["missing_expression"]
-        ds_1_domain = data_source_1.domain
         table_manager = ConfigurableReportTableManager([DynamicDataSourceProvider()])
         table_manager.bootstrap()
-        self.assertEqual(0, len(table_manager.table_adapters_by_domain))
-        self.assertEqual(0, len(table_manager.table_adapters_by_domain[ds_1_domain]))
```

Comment:
how did that work beforehand? Was there any point in the second check?

Reply:
I don't think there was any point. `table_manager.table_adapters_by_domain` is a `defaultdict`, so it will not raise `KeyError` when a missing item is retrieved (although that adds an item). However, these assertions would have failed if done in the opposite order.